### ISSUE TITLE
Add warning text for multi-allelic ExAC alleles in the Population tab of the VCI

### ISSUE THREAD

**Issue by wrightmw** [ClinGen/clincoded#1358]
Multi-allelic ExAC loci may be displayed incorrectly in the VCI, this will be fixed after the next myvariant.info release in mid-June. Until then we should not return data for multi-allelic loci (i.e. those ones returned as arrays) and instead display a warning message.

Something along the lines of:
We are not currently displaying ExAC data for multi-allelic loci because sometimes the data returned is for the alternate allele at the same locus. This issue will be fixed soon. Search ExAC for this variant.

**Comment by wrightmw** [comment]
Example for testing: ClinVar VariationID: 592

**Comment by jimmyzhen** [comment]
This ticket is addressed in the #1317 branch.

**Comment by jimmyzhen** [comment]
Hi @selinad, @wrightmw,

You may review the changes pertinent to this ticket in the following instance:
https://1317-jz-hpo-links-f4feeba-jzhen.demo.clinicalgenome.org

Please let me know if you have any questions. Thanks.

**Comment by selinad** [comment]
Slight change to this text:
"We are not currently displaying ExAC data for multi-allelic loci because sometimes the data returned is for the alternate allele at the same locus. This issue will be fixed soon. For the correct allele, see data in ExAC."

@wrightmw how about this:

"ExAC data is not currently displayed for this variant as it occurs at a multi-allelic locus and data is sometimes returned for an alternate allele due to changes in the data structure.  This issue is being addressed and will be fixed soon.  For the correct allele, see data in ExAC."

**Comment by wrightmw** [comment]
@jimmyzhen The functionality of this is great. Please feel free to amend to Selina's text (above).

**Comment by selinad** [comment]
@jimmyzhen I am not seeing the note and am seeing ExAC data for CV 592: https://r11-rc1-ebe540c-jzhen.demo.clinicalgenome.org/variant-central/?variant=488a886b-1525-4f0d-a38e-b73d1f594ec3&tab=population

**Comment by selinad** [comment]
@jimmyzhen and I sleuthed this out and figured out that MyVariant.info updated its file so it's no longer an array!  This is why @jimmyzhen 's message doesn't display (code looks for the array).  We checked the data for CV IDs 592 and 21145 and all looked perfect (corresponded with ExAC).  

Additionally, we looked at the other allele (CV 7716) for 21145 and that was correct.  

@jimmyzhen will leave code in, but the MyVariant.info fix is magical!  🦄
